build/protobuf.mk

```diff
@@ -17,15 +17,14 @@
 # This file is evaluated in the repo's parent directory. See main.go's
 # go:generate invocation.
 
-ORG_ROOT       := .
-REPO_ROOT      := $(ORG_ROOT)/cockroach
-GITHUB_ROOT    := $(ORG_ROOT)/..
+REPO_ROOT      := ./cockroach
+GITHUB_ROOT    := $(REPO_ROOT)/vendor/github.com
```

Comment:
In a repo with vendored sources, it seems like we probably don't need to keep the `executed in ..` thing either?